### PR TITLE
Update the WP packages with fixes prior to WP 6.2 beta 2

### PR DESCRIPTION
**Pull request by ntsekouras**
Trac ticket: https://core.trac.wordpress.org/ticket/57471

This updated the packages to the latest versions with the latest fixes.

https://github.com/WordPress/gutenberg/commit/1bf01c01a8238ce3a681ad1e517f86033818b78d

Includes the following changes:

- Add missing period in keyboard shortcut descriptions
- Site Editor: Append template type and name to the site editor page title
- Specify active color state for template navigation button
- Global Styles: Move more link about custom CSS to part of description
- ToolsPanel: Display optional items when values are updated externally
- File block: Re-add editor styles for classic themes
- Enable undo after creating a new menu
- Style Book: Exclude blocks that are not allowed to insert
- Add basic tsconfig.json validation
- Fix Experiments native variant breakage
- Update moment and moment-timezone packages to fix timezone issues
- NavigatorButton: Reuse Button types
- ToolsPanel: Ensure display of optional items when panel id is null
- ColorPalette: Ensure text label contrast checking works with CSS variables
- Components: Add support for named arguments in the navigator components
- Rename experiments package to private-apis
- Cherry-pick CLI: Fix the default label to match the documentation
- Handle block metadata attribute and related experimental APIs
- Fix: Remove browser default border for iframe in the editor
- [Layout]: Fix align controls for hybrid themes
- Visual Regression tests: use default playwright utils
- SelectControl: Fix multiple prop styling
- Fix clicking on the toggle button not closing the block inserter
- Update deps for the useEffect that creates navigation menus
- Minor updates in Private APIs docs
- Site Editor: Fix custom Template Parts rename action
- [Block Library - Cover]: Ensure url is not malformed due to sanitization through wp_kses
- Revert "Prevent the image from being resized larger than its container"
- Lodash: Remove from @wordpress/keycodes package
- Distraction free mode: Fix keyboard shortcut not working
- Popover: lock the __experimentalPopoverPositionToPlacement function
- Rename the "experiments" export to "privateApis"
- Add an aria label to the site save dialog
- Navigator: add more pattern matching tests, refine existing tests
- Add parent navigation support for the navigator component
- Add back link to Design heading in site editor navigation to return to Dashboard
- Add a nested level when selecting templates or template parts
- Move site editor 6.2 specific code to the right file
- Fix site editor navigation

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/site-editor.php [hunk 1]
```diff
@@ -28,23 +28,6 @@
...
-	if ( ! empty( $_GET['styles'] ) ) {
-		$home_template['styles'] = sanitize_key( $_GET['styles'] );
-	}
-	$redirect_url = add_query_arg(
-		$home_template,
-		admin_url( 'site-editor.php' )
-	);
-	wp_safe_redirect( $redirect_url );
-	exit;
-}
```
Comment (youknowriad) [inline src/wp-admin/site-editor.php]: yep this makes sense 👍

Comment (hellofromtonya) [reply src/wp-admin/site-editor.php]: Is this change part of the `@wordpress` package updates? Thinking it's a separate backport.

Comment (hellofromtonya) [reply src/wp-admin/site-editor.php]: Also not seeing this change in https://github.com/WordPress/gutenberg/commit/1bf01c01a8238ce3a681ad1e517f86033818b78d. Curious where and why it's being made.

Comment (ntsekouras) [reply src/wp-admin/site-editor.php]: It's a back port from https://github.com/WordPress/gutenberg/pull/48023, which is included in the packages update.

**Comment by hellofromtonya** [comment]
Currently reviewing.

File: src/wp-admin/site-editor.php (hunk 1 above)
Comment (hellofromtonya) [reply src/wp-admin/site-editor.php]: In talking with @ntsekouras, the PHP change here is separate from the scope of updating the `@wordpress` packages in Core. Nik will revert this PHP change and open a separate backport Trac ticket and PR for it. I'll get both reviewed and committed shortly.

Comment (ntsekouras) [reply src/wp-admin/site-editor.php]: I removed the commit and will open a new trac ticket for it and a separate PR

**Review by hellofromtonya — APPROVED** [review]
* Package version numbers match https://github.com/WordPress/gutenberg/commit/1bf01c01a8238ce3a681ad1e517f86033818b78d ✅ 
* CI is passing ✅ 
* Local testing - no errors ✅ 

Ready for commit 👍

**Review by felixarntz — APPROVED** [review]
@ntsekouras LGTM!

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/55333.

**Pull request closed without merging**